tests/functional/sync_v3_sign_in.js

```diff
@@ -232,7 +232,7 @@ registerSuite('Firefox Desktop Sync v3 signin', {
 
 registerSuite('Firefox Desktop Sync v3 signin - token code', {
   beforeEach: function () {
-    email = TestHelpers.createEmail();
+    email = TestHelpers.createEmail('sync{id}');
```

Comment:
Updated to use a sync email here because those emails are always required to perform a verification.